code/modules/materials/materials.dm

```diff
@@ -396,7 +393,7 @@ var/list/name_to_material
 	conductivity = 13 // For the purposes of balance.
 	stack_origin_tech = list(TECH_MATERIAL = 2)
 	composite_material = list(DEFAULT_WALL_MATERIAL = SHEET_MATERIAL_AMOUNT, "platinum" = SHEET_MATERIAL_AMOUNT) //todo
-	radiation_resistance = 60 //Plasteel is presumably dense and is the dominant material used in the engine. Still not great.

```

Comment:
Isn't this a big downgrade to plasteel's radiation protection since the weight is only 23? Also durasteel below? Maybe I'm just not seeing why it's equivalent.

Reply:
This is a return to what it was originally, with 60 it blocks nearly all radiation from a normal supermatter engine.

Reply:
On our smaller map, is that safe? Given it may or may not just irradiate the entire z-level while running?

Reply:
It is, this returns it to what it has been since the map was made and what everything is designed around. At the higher protection level radiation might as well not exist.